Lukem/index page node (#214)
moondance-labs/tanssi-docs: node-operators/sequencers/onboarding/index.md

---
title: Onboard to Run a Sequencer Node
description: Learn how to get started to run a Sequencer (also known as producer nodes) using Docker or Systemd to participate in the protocol.
icon: octicons-arrow-up-right-24
template: index-page.html
---

# Onboarding

Starting your journey as a sequencer begins here. This section covers the foundational steps to set up your node and prepare your account for active participation in the Tanssi network.

> Getting Started as a Sequencer:

Reviewer: ```suggestion
Getting started as a sequencer:
```

> - **Hardware requirements** - understand the recommended hardware specifications to ensure optimal performance of your [sequencer](run-a-sequencer/index.md) node

Reviewer: ```suggestion
- **Hardware requirements** - understand the recommended hardware specifications to ensure optimal performance of your [sequencer](/node-operators/sequencers/onboarding/run-a-sequencer/) node
```

- **Port configuration** - learn which network ports need to be open to facilitate proper communication within the Tanssi ecosystem
- **Session keys setup** - detailed instructions on generating and managing session keys, a vital step for enabling block production capabilities

> - **Docker and systemd deployment** - Choose between Docker and systemd for deploying your node, with step-by-step guides tailored for each method

Reviewer: ```suggestion
- **Docker and systemd deployment** - choose between Docker and systemd for deploying your node, with step-by-step guides tailored for each method
```
